cmd/node/flags.go

```diff
@@ -322,6 +322,12 @@ var (
 			"and by advanced users, as a too high memory ballast could lead to Out Of Memory panics. The memory ballast " +
 			"should not be higher than 20-25% of the machine's available RAM",
 	}
+
+	// autoGenerateSigningKey defines a flag that, if set, will generate every time when node starts a new signing key
+	autoGenerateSigningKey = cli.BoolFlag{
+		Name:  "auto-generate-signing-key",
```

Comment:
very nice and helpful PR ! :rocket: 
can we make the flag shorter? It's not something one can remember easily. can we try `--no-key` or something similar?
@iulianpascalau ideas?

Reply:
yup, sounds good something shorter
Maybe `--autogenerate-key` ?

Reply:
changed

Reply:
the correct form should be `--auto-generate-key`. 
but an even better naming would be `--new-key` which I think it's pretty self-explanatory (and also easy to remember)